src/main/java/pl/nogacz/snake/board/Board.java

```diff
@@ -22,16 +23,19 @@
     private Design design;
     private Random random = new Random();
 
-    private boolean isEndGame = false;
+    public boolean isEndGame = false, thereIsFood = false, thereIsItem = false;
 
     private static int direction = 1; // 1 - UP || 2 - BOTTOM || 3 - LEFT || 4 - RIGHT
     private int tailLength = 0;
+    public int counter = -1, itemTimer = -1;

```

Comment:
Accessor metotları tanımla. 

Değişken isimleri yine çok genel olmuş. Senin eklediğin özelliğe spesifik isimler olsun. Counter neyi sayıyor? Herşeyi saymadığına göre daha spesifik ne saydığını ismini değiştirerek belirt. Aynı şekilde itemTimer hangi itemın timerı. Ayrıca bir timer neden int olarak tanımlı? Bu da mı bir sayaç? Ya da bunu long olarak mı tanımlamak istemiştin eğer bu bir millisaniye gibi bir şey ise?

İsimleri daha doğru seçersen bu tür kafa karışıklıkları daha aza iner.

Reply:
Done.